charts/internal/machine-class/values.yaml

```diff
@@ -20,9 +20,14 @@ machineClasses:
   dnsConfig:
     nameservers:
     - 8.8.8.8
-  memoryFeatures:
+  memory:
```

Comment:
You already have a field named `memory` above. I believe `cpus` and that old `memory` field are no longer needed and should be replaced by `resources`?